spec/tail_spec.rb

```diff
@@ -146,6 +147,7 @@
     let(:directory) { Stud::Temporary.directory }
     let(:file_path) { File.join(directory, "1.log") }
     let(:position) { :beginning }
+    let(:should_yield) { [[file_path, "line1"], [file_path, "line2"]] }
```

Comment:
I agree but I would need to reformat the expected array so it does not span off to the RHS too far.